src/ucp/proto/proto_common.h

```diff
@@ -11,14 +11,22 @@
 #include "proto_select.h"
 
 
+/* Constant for "undefined"/"not-applicable" structure offset */
+#define UCP_PROTO_COMMON_OFFSET_INVALID          PTRDIFF_MAX
+
+
 typedef enum {
     UCP_PROTO_COMMON_INIT_FLAG_SEND_ZCOPY    = UCS_BIT(0), /* Send buffer is used by
                                                               zero-copy operations */
     UCP_PROTO_COMMON_INIT_FLAG_RECV_ZCOPY    = UCS_BIT(1), /* Receive side is not
                                                               doing memory copy */
     UCP_PROTO_COMMON_INIT_FLAG_REMOTE_ACCESS = UCS_BIT(2), /* One-sided remote access */
-    UCP_PROTO_COMMON_INIT_FLAG_MEM_TYPE      = UCS_BIT(3)  /* Protocol supports
+    UCP_PROTO_COMMON_INIT_FLAG_MEM_TYPE      = UCS_BIT(3), /* Protocol supports
                                                               non-host buffers */
+    UCP_PROTO_COMMON_INIT_FLAG_RESPONSE      = UCS_BIT(4), /* Sender side has to get
```

Comment:
How it is different from send zcopy ?

Reply:
It means the buck data transfer is don from target->initiator so no need to consider fragment size on sender side.

Reply:
in send_zcopy without flag FAST_COMPLETION would estimate the latency from sender to receiver, but in get_zcopy need to measure the latency until data arrives back (including both the request and the response)